README.md

```diff
@@ -53,65 +53,83 @@ To use an SDK client, instantiate an object of that client and configure these p
  * Sample TypeScript code that shows how Commerce SDK can access Salesforce Commerce
  * APIs.
  * 
- * To learn more about the parameters please refer to https://developer.salesforce.com/docs/commerce/commerce-api/guide/get-started.html
+ * For more information, see (Get started with Salesforce Commerce B2C APIs)https://developer.salesforce.com/docs/commerce/commerce-api/guide/get-started.html
  */
-​
+
 // Import the SDK in TypeScript
 // tsc requires the --esModuleInterop flag for this
-import * as CommerceSdk from "commerce-sdk";
+import { Search, Customer } from "commerce-sdk";
 // For Javascript, use:
 // import CommerceSdk from "commerce-sdk";
-const { ClientConfig, helpers, Search } = CommerceSdk;
+// const { Search, Customer } = CommerceSdk;
 // Older Node.js versions can instead use:
 // const { ClientConfig, helpers, Search } = require("commerce-sdk");
 
-// Create a configuration to use when creating API clients
+// demo client credentials, if you have access to your own please replace them below.
+// do not store client secret as plaintext. Store it in a secure location.
+const CLIENT_ID = "da422690-7800-41d1-8ee4-3ce983961078";
+const CLIENT_SECRET = "D*HHUrgO2%qADp2JTIUi";
+const ORG_ID = "f_ecom_zzte_053";
+const SHORT_CODE = "kv7kzm78";
+const SITE_ID = "RefArch";
+
+// client configuration parameters
 const config = {
-    headers: {},
-    parameters: {
-      clientId: '<your-client-id>',
-      organizationId: '<your-org-id>',
-      shortCode: '<your-short-code>',
-      siteId: '<your-site-id>'
-    }
+  headers: {},
+  parameters: {
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
+  },
+};
+
+/**
```

Comment:
Can't suggest across the whole block, but here's a snippet just to tidy up and save a few lines.
```ts
const credentials = Buffer.from(`${CLIENT_ID}:${CLIENT_SECRET}`).toString("base64");
const loginClient = new Customer.ShopperLogin({
  // Headers can be specified in the client config and used for all requests
  headers: { authorization: `Basic ${credentials}` },
  ...config
});
// Get the shopper or guest JWT/access token, along with a refresh token, using client credentials
const token = await loginClient.getAccessToken({
  headers,
  body: { grant_type: "client_credentials" },
});

const searchClient = new Search.ShopperSearch(config);
// Search for dresses
const searchResults = await searchClient.productSearch({
  // Headers can also be provided per request
  headers: { authorization: `Bearer ${token.access_token}` },
  parameters: { q: "dress", limit: 5 }
});

if (searchResults.total) {
  const firstResult = searchResults.hits[0];
  console.log(`${firstResult.productId} ${firstResult.productName}`);
} else {
  console.log("No results for search.");
}
```